docs/source/api/deliveryservices_id_servers_eligible.rst

```diff
@@ -27,7 +27,7 @@ Retrieves properties of Edge-Tier servers eligible for assignment to a particula
 
 1. Server Type: Edge, Origin
 2. CDN: Server, :term:`Delivery Service` must belong to the same CDN
-3. Server Capabilities: If the :term:`Delivery Service` has  :term:`Delivery Service required capabilities`, a server must have all defined capabilities
+3. Server Capabilities: If the :term:`Delivery Service` has  :term:`Delivery Service required capabilities`, an edge server must have all defined capabilities
```

Comment:
should this be origin server?

Reply:
No, because mids can't be assigned to Delivery Services directly. So only Edges need to be called out here.

Reply:
technically i think mids can be assigned directly to ds's but it will have no effect.